sweyer_plugin/lib/sweyer_plugin.dart

```diff
@@ -0,0 +1,183 @@
+import 'dart:typed_data';
+import 'dart:ui';
+
+import 'package:equatable/equatable.dart';
+import 'package:sweyer_plugin/sweyer_plugin_method_channel.dart';
+import 'package:uuid/uuid.dart';
+
+import 'sweyer_plugin_platform_interface.dart';
+
+/// A song from the native media store.
+abstract class MediaStoreSong {
```

Comment:
If we are abstracting out from the MediaStore, wouldn't it make more sense to call this something like RawSong or AbstractSong?

Here and for other classes

Reply:
Yeah, let's rename it. I would prefer `PlatformSong`, because this is the representation of the song coming from the platform we are running on. Is that ok?

Reply:
Yeah